SwrveConversationSDK/Conversation/SwrveConversationAtomFactory.m

```diff
@@ -27,7 +27,6 @@ @implementation SwrveConversationAtomFactory
     
     if([type isEqualToString:kSwrveContentTypeHTML]) {
         SwrveContentHTML *swrveContentHTML = [[SwrveContentHTML alloc] initWithTag:tag andDictionary:dict];
-        swrveContentHTML.style = [dict objectForKey:kSwrveKeyStyle];
```

Comment:
The style is stored as part of the "initWithTag:tag andDictionary:dict" method:
SwrveContentHTML *swrveContentHTML = [[SwrveContentHTML alloc] initWithTag:tag andDictionary:dict];